requirements/pytorch/strategies.txt

```diff
@@ -1,7 +1,7 @@
 # NOTE: the upper bound for the package version is only set for CI stability, and it is dropped while installing this package
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
 
-colossalai>=0.1.10
+# colossalai>=0.1.10
```

Comment:
I would make an exception and keep this unless we actually start seeing broken master CI from this. If that happens and their devs are unable to help, then we comment the dependency until fixed.

Reply:
I see broken PR :otter: #15942

Reply:
```suggestion
# colossalai>=0.1.10  # TODO: uncomment when there's a stable version released
```